test/integration/content-sidebar/TaskForm.e2e.test.js

```diff
@@ -6,11 +6,10 @@ describe('Create Task', () => {
     const getMessageField = () => cy.getByTestId('task-form-name-input');
     const getSubmitButton = () => cy.getByTestId('task-form-submit-button');
     const getCancelButton = () => cy.getByTestId('task-form-cancel-button');
-    const username = 'Platform '; // will be used as assignee
+    const username = 'PreviewTestApp'; // will be used as assignee
 
     beforeEach(() => {
         cy.visit('/Elements/ContentSidebar'); // Open sidebar example page
-        cy.getByTestId('sidebaractivity').click(); // Open activity tab
```

Comment:
We're not using a skills file by default so the sidebar will start on the activity feed.

Reply:
Dont want to add some skills? :o

Reply:
Added skills to all videos on the folder, but our base test file is a document.